admin/src/components/SaveButton/SavePaperTrailButton.jsx

```
@@ -0,0 +1,88 @@
+ */
+const SavePaperTrailButton = ({ enabled }) => {
+  const [saving, setSaving] = useState(false);
+  const [saved, setSaved] = useState(false);
+  const { formatMessage } = useIntl();
+  
+  // Extract UID from URL params
+  const params = useParams();
+  let uid = '';
+  
+  // The URL format is different depending on whether it's a single type or collection type
+  if (params && params.uid) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)